<launch>
    <!-- <node pkg="dist_num" type="centralized.py" name="centralized_node" output="screen">
    </node> -->
    <node pkg="dist_num" type="publisher.py" name="publisher_node" output="screen">
    </node>
    <!-- <node pkg="dist_num" type="subscriber.py" name="subscriber_node" output="screen">
    </node> -->
</launch>